### PR TITLE
overlapping of text is handled by changing the orientation of text

### PR DESCRIPTION
**Pull request by amanvar**
continuation of pull request #688 

when you open it in google chrome web browser there is some overlapping of text is found(in firefox it works fine).
I changed it by changing the orientation of text.
overlapping removed.

### CODE REVIEW

**Comment by jobara** [comment]
@amanvar could you update the pull request name and the commit log to comply with the [coding and commit standards](https://wiki.fluidproject.org/display/fluid/Coding+and+Commit+Standards).

GitHub has information on how to amend a commit message.
https://help.github.com/articles/changing-a-commit-message/

File: demos/uiOptions/css/uiOptions.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 
 body {
-    font-family: LatoLight, Helvetica, Tahoma, Verdana, sans-serif;
+    font-family: LatoLight, Helvetica, Tahoma, Verdana, sans-serif;z
```
Comment (jobara) [inline demos/uiOptions/css/uiOptions.css]: typo "z" at the end of the line

**Comment by amanvar** [comment]
@jobara  I updated the PR.
Thanks

**Comment by jobara** [comment]
@amanvar thanks for sending me the CLA.

Additionally as per this comment ( https://github.com/fluid-project/infusion/pull/692#issuecomment-198342007 ) could you update your pull request. The commit logs and the Pull Request itself should all contain the JIRA issue number at the beginning. This is so that we can keep track of what the work is related to.

**Comment by cindyli** [comment]
@amanvar, do you plan to continue working on this pull request? If not, this pull request will be closed due to its inactivity.

**Comment by cindyli** [comment]
Closed as an inactive pull request.

**Pull request closed without merging**